src/out/thirdparty/scanneropen_file.check.c

```diff
@@ -0,0 +1,28 @@
+/*****************************************
+Emitting C Generated Code
+*******************************************/
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE
+#endif
+#include <stdlib.h>
+#include <scanner_header.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <stdbool.h>
+/**************** Snippet ****************/
+void Snippet(int x0) {
+  int x1 = open("/scratch-ml00/wang603/lms-umbrella/lms-clean/src/test/scala/lms/thirdparty/test_scanner.scala", 0);
```

Comment:
this hard-coded path is wrong 🎃 

idk how to make this not hard-coded. 
maybe just don't check with files?